jenkins/scripts/perf/local/submit.py

```
@@ -342,6 +346,26 @@ def partition_has_gpu_gres(partition):
         return False
 
 
+def detect_cluster_name():
+    """Best-effort Slurm cluster name detection on the submission frontend."""
+    name = os.environ.get("SLURM_CLUSTER_NAME", "")
+    if name:
+        return name
+    try:
+        config_out = subprocess.check_output(
+            ["scontrol", "show", "config"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)